Makefile

```diff
@@ -17,8 +17,15 @@ VERSION:=$(shell awk -F\' '/const VERSION/ {print $$2}' < src/DDTrace/Tracer.php
 
 INI_FILE := $(shell php -i | awk -F"=>" '/Scan this dir for additional .ini files/ {print $$2}')/ddtrace.ini
 
+RUN_TESTS_IS_PARALLEL := $(shell test $(shell php-config --vernum) -gt 70399 && echo 1 || echo 0)
+
+ifeq ($(RUN_TESTS_IS_PARALLEL), 1)
 RUN_TESTS_EXTRA_ARGS := -j$(shell nproc)
-RUN_TESTS_CMD := REPORT_EXIT_STATUS=1 TEST_PHP_SRCDIR=$(PROJECT_ROOT) USE_TRACKED_ALLOC=1 php -n -d 'memory_limit=-1' $(BUILD_DIR)/run-tests.php --show-diff -n -p $(shell which php) -q $(RUN_TESTS_EXTRA_ARGS)
+else
+RUN_TESTS_EXTRA_ARGS :=
+endif
+
+RUN_TESTS_CMD := RUN_TESTS_IS_PARALLEL=$(RUN_TESTS_IS_PARALLEL) REPORT_EXIT_STATUS=1 TEST_PHP_SRCDIR=$(PROJECT_ROOT) USE_TRACKED_ALLOC=1 php -n -d 'memory_limit=-1' $(BUILD_DIR)/run-tests.php --show-diff -n -p $(shell which php) -q $(RUN_TESTS_EXTRA_ARGS)
```

Comment:
This (RUN_TESTS_IS_PARALLEL in env) is just for debug in CI run, I'll remove it ...